packages/coinstac-ui/webpack.config.js

```diff
@@ -115,22 +117,13 @@ if (process.env.NODE_ENV === 'development') {
   config.bail = false;
   // config.plugins.push(new webpack.NoErrorsPlugin());
 
-  config.devServer = {
-    inline: true,
-  };
-
   // Massage configuration for hot module replacement:
   config.output.publicPath = `http://localhost:${port}/`;
   config.plugins.push(
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NamedModulesPlugin()
   );
 
-  config.entry.unshift(
```

Comment:
Newer versions of webpack-dev-server and the hot module replacement include these clients in the bundle automatically. _I think._